sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -332,7 +332,7 @@ resource
     ;
 
 queryNoWith
-    : insertInto? queryTerm queryOrganization                                              #singleInsertQuery
+    : (insertInto (hints+=hint)*)? queryTerm queryOrganization                             #singleInsertQuery
```

Comment:
Why allow more than one coalesce hint? Is this useful in some way?

I think it is confusing to users to allow multiple hints because the order in which they are applied to the plan isn't clear. Is it left-to-right order? Would `REPARTITION(100), COALESCE(10)` repartition and then coalesce?